pom.xml

```diff
@@ -284,7 +284,26 @@
       <id>validator</id>
       <build>
         <plugins>
-
+          <plugin>
+            <artifactId>maven-antrun-plugin</artifactId>
+            <version>1.7</version>
+            <executions>
+              <execution>
+                <id>copy-config</id>
```

Comment:
Unless you think otherwise, it would be good to rename this ID to something more meaningful.
Since those are shown in the build log, it's nice to see something more descriptive.

Could you rename to:
> Copy OpenMRS Initializer configs for validation

Reply:
Could you also rename below:
```
<id>validate-configurations</id>
```
into
```
<id>Validate OpenMRS Initializer configs</id>
```

Reply:
Correct, thanks. Resolved.